Formula/llvm@11.rb

```diff
@@ -167,6 +190,14 @@ def install
       system "cmake", "--build", ".", "--target", "install-xcode-toolchain" if MacOS::Xcode.installed?
     end
 
+    on_linux do
+      # Strip executables/libraries/object files to reduce their size
+      system("strip", "--strip-unneeded", "--preserve-dates", *(Dir[bin/"**/*", lib/"**/*"]).select do |f|
+        f = Pathname.new(f)
+        f.file? && (f.elf? || f.extname == ".a")
+      end)
+    end
```

Comment:
Their build scripts generate an `install/strip` target. Wouldn't it be better to call that target instead on Linux? Or maybe calling it on both, though I'd want to know what this changes on macOS.

Reply:
I think this might be carried over from the older versions where we weren't using the monorepo to build.  I'll try to test this locally and see if it attains the same result on Linux.  On macOS I can call it as well, and if it passes all the tests fine then maybe it's okay to use on both?

Reply:
Sounds good. We can also compare the existing bottle to the newly built one to see if something looks off.

Reply:
I built LLVM 12 locally with `install/strip` as the target and the bottle size only differs by 7 MB between the version built with `install/strip` and the original one we manually stripped.  I also check some of the binaries produced by `install/strip` and `file` reports them all to be stripped.

Is there anything else we should be checking for comparison?  I'm happy to build LLVM 11 and 12 bottles in the linuxbrew/core CI so that the artifacts can be downloaded to be compared if that would be useful.